3-mcstas/make_powder_instrument.py

```diff
@@ -538,6 +558,8 @@ def make(
             priority=i + 5,  # Ensure unique priorities
             number_of_activations=f"{option}_active",
         )
+        if i == len(options)-1:
+            sample.set_SPLIT(100)
```

Comment:
This code adds split to a specific sample, but the rays will be split at that point in the instrument file regardless of this sample being active or not. We might as well just set a split on Union_master that is applied for all options, we can later tailer the split number to be specific to each sample if necessary.